src/libraries/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <DefineConstants>$(DefineConstants);NETSTANDARD</DefineConstants>
     <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
+    <NoWarn>$(NoWarn);SYSLIB0014</NoWarn>
```

Comment:
Suggestion: add a comment indicating what `SYSLIB0014` is. We tried to do that with the other `SYSLIB` obsoletions.

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Reflection.MetadataLoadContext/tests/System.Reflection.MetadataLoadContext.Tests.csproj#L5-L6